### DISCUSSION TITLE
Parquet Sharp 8.0.0-beta1

### DISCUSSION THREAD

**Discussion by kaleshanagineni** [G-Research/ParquetSharp#305] [Q&A] [answered]
Hi, 
We are currently using Parquet Sharp in our project. We are using version 8.0.0-beta1 as it has method in ParquestFile.CreateParquetRowWriter which accept Columns[] for one of our uses. we are in verge releasing our product. we would like to know when stable version 8.0.0 will be released ? may I know if you have any anticipated release date ?

**Comment by adamreeve** [comment, marked as answer]
Hi @kaleshanagineni, we don't have a planned date for the release yet. We want to wait a little more time to allow testing of the latest 8.0.0-beta2 and then assuming we don't hear about any bugs we'll make the final 8.0.0 release within the next couple of weeks.

**Reply by kaleshanagineni** [reply]
Hi @adamreeve 
Thanks for your response. do you have any scheduled release time on this now?

**Reply by jgiannuzzi** [reply]
Hi @kaleshanagineni, we have just released [8.0.0](https://github.com/G-Research/ParquetSharp/releases/tag/8.0.0).
Enjoy, and good luck with your product release!

**Reply by kaleshanagineni** [reply]
Thanks @jgiannuzzi.
